CrossprojectpipingExternalModule.php

```diff
@@ -971,9 +971,14 @@ function pipeToRecord($dst_rid) {
 						) {
 							continue;
 						}
-						
+
+						// get the destination project's name for this source pipe field
+						$pipe_field_index = array_search($field_name, $src_project['source_fields'], true);
+						$dst_name = $src_project['dest_fields'][$pipe_field_index];
+
 						// skip this field if the destination record's form status for the containing form is above the pipe limit
-						$form_name = $src_project['dest_forms_by_field_name'][$field_name];
+						$form_name = $src_project['dest_forms_by_field_name'][$dst_name];
```

Comment:
`$src_project['dest_forms_by_field_name']` is an array and contains index for pipe fields only.
Currently `$form_name` is the unique identity field which will result in empty string.

Reply:
This changes looks great!  Resolving...